src/Tribe/Aggregator/Tabs/New.php

```diff
@@ -284,6 +284,14 @@ public function get_result_messages( $queue ) {
 				);
 			}
 
+			$images_scheduled = $queue->activity->get( 'images', 'scheduled' );
+			if ( ! empty( $images_scheduled ) ) {
+				$messages['success'][] = sprintf( // add image import count
+						_n( '%1$d new image was scheduled for import.', '%1$d new images were scheduled for import.', $queue->activity->count( 'images', 'scheduled' ), 'the-events-calendar' ),
+						$queue->activity->count( 'images', 'scheduled' )
```

Comment:
:no_entry_sign: Multi-line function call not indented correctly; expected 20 spaces but found 24

<sub>hash: a825c365ab2196641bd6cce769d74522</sub>